### ISSUE TITLE
Really slow after last update

### ISSUE THREAD

**Issue by eduardoroliveira** [microsoft/vscode#165438]
Type: <b>Bug</b>

This has been just updated but everything is toooooooooo slow now. Not able to use right now!

VS Code version: Code 1.73.0 (8fa188b2b301d36553cbc9ce1b0a146ccb93351f, 2022-11-01T15:38:50.881Z)
OS version: Darwin x64 21.6.0
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz (12 x 2200)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|6, 7, 6|
|Memory (System)|32.00GB (0.02GB free)|
|Process Argv|--crash-reporter-id 294faef8-4a6a-4bc0-931f-7d0000ca47bc|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (56)</summary>

Extension|Author (truncated)|Version
---|---|---
simple-react-snippets|bur|1.2.7
doxdocgen|csc|1.4.0
vscode-markdownlint|Dav|0.48.1
vscode-eslint|dba|2.2.6
githistory|don|0.6.19
EditorConfig|Edi|0.16.4
vsc-material-theme-icons|equ|2.4.0
terraform-autocomplete|erd|0.0.8
prettier-vscode|esb|9.9.0
go|gol|0.35.2
cloudcode|goo|1.20.3
vscode-graphql|Gra|0.8.1
vscode-graphql-execution|Gra|0.1.5
vscode-graphql-syntax|Gra|1.0.4
terraform|has|2.24.3
better-cpp-syntax|jef|1.16.3
cmake-language-support-vscode|jos|0.0.7
file-downloader|min|1.0.12
mindaro|min|1.0.120221031
vscode-docker|ms-|1.22.2
vscode-dotnet-runtime|ms-|1.6.0
vscode-kubernetes-tools|ms-|1.3.10
playwright|ms-|0.2.13
isort|ms-|2022.4.0
python|ms-|2022.18.1
vscode-pylance|ms-|2022.11.10
jupyter|ms-|2022.9.1202862440
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.12
vscode-jupyter-cell-tags|ms-|0.1.6
vscode-jupyter-slideshow|ms-|0.1.5
remote-containers|ms-|0.262.3
remote-ssh|ms-|0.90.1
remote-wsl|ms-|0.72.0
cmake-tools|ms-|1.12.27
cpptools|ms-|1.12.4
cpptools-extension-pack|ms-|1.3.0
vscode-typescript-next|ms-|5.0.20221102
vscode-typescript-tslint-plugin|ms-|1.3.4
vscode-ginkgo|ons|0.0.1
java|red|1.12.0
vscode-commons|red|0.0.6
vscode-yaml|red|1.10.1
markdown-preview-enhanced|shd|0.6.5
vscode-stylelint|sty|1.2.3
cmake|twx|0.0.17
intellicode-api-usage-examples|Vis|0.2.6
vscodeintellicode|Vis|1.2.29
vscode-java-debug|vsc|0.46.0
vscode-java-dependency|vsc|0.21.1
vscode-java-pack|vsc|0.25.6
vscode-java-test|vsc|0.37.1
vscode-maven|vsc|0.39.2
vscode-icons|vsc|12.0.1
vscode-todo-highlight|way|1.0.5
vscode-proto3|zxh|0.5.5

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscoreces:30445986
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593:30376534
pythonvs932:30410667
cppdebug:30492333
vsclangdc:30486549
c4g48928:30535728
dsvsc012:30540252
azure-dev_surveyone:30548225
pyindex848:30577860
nodejswelcome1cf:30587006
fc301958:30595537
282f8724:30602487
gswce1:30602634
iaj6b796:30602301

```

</details>

**Comment by roblourens** [comment]
Please give some more details about exactly what you are doing

**Comment by jgoz** [comment]
Not many details here, but I noticed this as well. 1.73.0 introduced really high latency when working with TypeScript, across the board. Input latency, speed of autocomplete, refactoring and navigation operations, etc. Downgrading to 1.72.2 restored the previous performance profile for me.

**Comment by Uninen** [comment]
I also experienced problems when saving TS files w/ 1.73.0. The editor would go into a what seemed like infinite loop of eslint import / indentation tweaks and saving files would either take several seconds or never finish. Tried to disable various extensions with no luck. Downgrading to 1.72 fixed the issue.

I spent an hour trying to find any log or debug messages relating to the issue and tweaking settings and extensions, found nothing and didn't figure out anything but the downgrading that helped.
